config/nephe.yml

```diff
@@ -473,6 +488,26 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - crd.antrea.io
+  resources:
+  - externalnodes
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
please add only the required permission for external node. we dont watch external node,

Reply:
Sure